### ISSUE TITLE
Sync alias pages

### ISSUE THREAD

**Issue by blueskyson** [tldr-pages/tldr#6599]
A few translation of alias pages is completed in [alias-pages.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).

Do we need to make a `set-alias-page.py` to read from English pages and sync translations according to alias-page.md?

**Comment by pixelcmtd** [comment]
This has been a TODO for my own tldr tooling for a long time, but I was too lazy to implement parsing the Markdown.

**Comment by marchersimon** [comment]
@blueskyson sounds like a great idea. I'm just not sure if it would be worth the effort, because this will be used once and then hardly ever again.

**Comment by blueskyson** [comment]
It doesn't really take effort, just copy and alter some essential blocks from `set-information-link.py`.

**Comment by marchersimon** [comment]
Alright, then let's do this!

Just something to think about: Should we make alias pages for a language only if the original command exists in that language? On the first glance this would make sense, but when you try to view e.g. the German page for `platformio` it displays the English page because we don't have the original command `pio` in German. But if we at least had `platformio` in German, that would at least help a bit.
tldr: I think we should translate all alias pages, even tough the original command doesn't exist in that language.

**Comment by CleanMachine1** [comment]
Can we close this issue?

**Comment by marchersimon** [comment]
I'd say we should wait for #6846
